client/src/app/site/motions/components/motion-detail/motion-detail.component.html

```diff
@@ -360,6 +360,20 @@ <h4 translate>Origin</h4>
     >
         <!-- Line Number and Diff buttons -->
         <div *ngIf="!editMotion && !motion.isStatuteAmendment()" class="motion-text-controls">
+            <mat-form-field class="motion-goto-line" *ngIf="highlightedLineOpened">
+                <input type="number" [(ngModel)]="highlightedLineTyping" [ngModelOptions]="{standalone: true}" matInput
```

Comment:
please add the [osAutoFocus](https://github.com/OpenSlides/OpenSlides/blob/master/client/src/app/shared/directives/autofocus.directive.ts) directive
looks misaligned. Please use [prettier](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAllADgVxgAhgE8s4BeAHRChwFsAjOAJyvwG0AKKAcwFkIAJnAA2ASgC6lEAAt03acLnT4AgDKY4AFRKZurNj35DhAeSwx00AM6SqwKzACGUAY+HQ4SQkxxwAvqy0jjAAkth47MxMEEwAyk7wvMFg0sy2MkqK8irqUHBJMCnMgZhSAIysWMKOYHDSEMJCTFLAwPgA5ADiEIQ9innt+AA+3s5W1fD4fgEgAHwgADQgEOaWUFbIoI5M0QDuAArbCBsojgBuEOgCiyD0TDUA1nAwsVg1usgwPnBLyrTCAHVZPArG9arFjugLGcoURkOArBslpgrMwYPt7twgsgAGZuVFLABWVgAHgAhe5gJ4vRy0OC5OC4-E-EDEkmxXTCOAARRwEHgTOEBJAbyYqKY8Oc3Bw1QlSyw0VRAPuWHhCrg4rOjKWTDgAEccOhdRjHFjHILhajaOhPt8llZOTy+QKkHihSynPQAVcYNJkAAWJZfRzofrcADCEFo2JQGoArDccKjNI56Cc3cKFZgYN6BL7kGUAEwABmmQA) to align HTML